forc-plugins/forc-client/src/util/pkg.rs

```diff
@@ -1,8 +1,142 @@
 use anyhow::Result;
 use forc_pkg::manifest::GenericManifestFile;
 use forc_pkg::{self as pkg, manifest::ManifestFile, BuildOpts, BuildPlan};
-use pkg::{build_with_options, BuiltPackage};
+use forc_util::user_forc_directory;
+use pkg::{build_with_options, BuiltPackage, PackageManifestFile};
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::PathBuf;
 use std::{collections::HashMap, path::Path, sync::Arc};
+use sway_utils::{MAIN_ENTRY, MANIFEST_FILE_NAME, SRC_DIR};
+
+/// The name of the folder that forc generated proxy contract project will reside at.
+pub const PROXY_CONTRACT_FOLDER_NAME: &str = ".generated_proxy_contracts";
+/// Forc.toml for the default proxy contract that 'generate_proxy_contract_src()' returns.
+pub const PROXY_CONTRACT_FORC_TOML: &str = r#"
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "proxy_contract"
+
+[dependencies]
+standards = { git = "https://github.com/FuelLabs/sway-standards/" }
```

Comment:
Should the version be pinned to avoid unplanned automatic upgrades?

Reply:
https://github.com/FuelLabs/sway-standards/issues/100 -- waiting on this

Reply:
I think due to urgency of this, we can continue as is. As soon as a release is cut on that PR we can make a hotfix. But i totatlly agree that we should pin this version so that breaking changes to the standard (i don't think that should be possible but in any case) won't break our proxy capabilities. cc @JoshuaBatty for visibility

Reply:
We can pin to the `rev` without needing to release. But it looks like you made a release 👍

Reply:
Yeah forgot that we had `rev` option 🤦. Updated to tag as we have it now!